lib/common/client/base.rb

```diff
@@ -112,7 +112,7 @@ def request(method, path, params = {}, headers = {}, options = {}) # rubocop:dis
       rescue Timeout::Error, Faraday::TimeoutError => e
         Raven.extra_context(service_name: config.service_name, url: path)
         raise Common::Exceptions::GatewayTimeout, e.class.name
-      rescue Faraday::ClientError => e
+      rescue Faraday::ClientError, Faraday::Error => e
```

Comment:
technically, this won't be necessary until we upgrade to faraday > 1.0 but there's no harm in adding it now.